lab_3_keywords_textrank/main.py

```diff
@@ -734,9 +878,31 @@ def train(self) -> int:
             int:
                 0 if importance scores were calculated successfully, otherwise -1
         """
-        pass
+        phrases = extract_phrases(self._text)
+        if phrases:
```

Comment:
revert and return -1 if not phrase. And apply this rule for every if in this file